app/Models/CsoActivityDomain.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Why is this necessary? The activity domain can just be a field right?

Reply:
The field for `domain of activity` are checkboxes. I thought it meant that multiple could be selected

Reply:
If only one `domain of activity` should be selected, it means maybe it has to be `radio` buttons instead of `checkbox`

Reply:
Okay